packages/exo/test/heap-classes.test.js

```diff
@@ -280,37 +283,41 @@ test('raw guards', t => {
       return 'hello';
     },
     rawIn(obj) {
-      t.is(Object.isFrozen(obj), false);
+      t.false(Object.isFrozen(obj));
```

Comment:
This test passes because the `M.raw` guard is bypassing the `harden` otherwise implied. It’s always passed an unfrozen/unhardened object in the tests.